tests/integration/test_create_update.py

```diff
@@ -66,7 +67,230 @@ def test_basic_bodhi_update(
             "f30-updates-testing": "sen-0.6.0-4.fc30",
             "f30-updates-testing-pending": "sen-0.6.0-4.fc30",
         },
-        save=lambda **kwargs: None,
+        save=lambda **kwargs: Munch(
+            {
```

Comment:
What about to have the `Munch` structure in the alone file and loaded before running the test.